engine/src/test/java/io/camunda/zeebe/engine/processing/processinstance/ModifyProcessInstanceRejectionTest.java

```diff
@@ -252,8 +252,9 @@ public void shouldRejectCommandWhenVariableScopeIsUnknown() {
         .hasRejectionType(RejectionType.INVALID_ARGUMENT)
         .hasRejectionReason(
             String.format(
-                "Expected to modify instance of process '%s' but it contains one or more variable "
-                    + "instructions with a scope element id that could not be found: 'C', 'D'",
+                """
+                Expected to modify instance of process '%s' but it contains one or more variable instructions \
+                with a scope element id that could not be found: 'C', 'D'""",
```

Comment:
🔧 I like these changes, but let's try not to exceed our own column limit of 100.

💭  Sadly, we have no way to automatically reflow strings in the IDEA plugin, which is why it isn't set in the spotless google-java-format config